enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -90,16 +90,48 @@ const (
 	// flushing any atime updates in an incomplete batch (that have not
 	// already been flushed due to throughput)
 	atimeFlushPeriod = 10 * time.Second
+
+	// Default values for options
+	defaultBlockCacheSizeBytes    = int64(1000 * megabyte)
+	defaultMaxInlineFileSizeBytes = int64(1024)
+	defaultAtimeUpdateThreshold   = 10 * time.Minute
+	defaultAtimeWriteBatchSize    = 1000
+	defaultAtimeBufferSize        = 100000
+	defaultMinEvictionAge         = 6 * time.Hour
 )
 
-// Options is a struct containing the pebble cache configuration options.
+// options is a struct containing the pebble cache configuration options.
 // Once a cache is created, the options may not be changed.
-type Options struct {
-	RootDirectory       string
-	Partitions          []disk.Partition
-	PartitionMappings   []disk.PartitionMapping
-	MaxSizeBytes        int64
-	BlockCacheSizeBytes int64
+type options struct {
+	RootDirectory     string
+	Partitions        []disk.Partition
+	PartitionMappings []disk.PartitionMapping
+
+	MaxSizeBytes           int64
+	BlockCacheSizeBytes    int64
+	MaxInlineFileSizeBytes int64
+
+	AtimeUpdateThreshold time.Duration
+	AtimeWriteBatchSize  int
+	AtimeBufferSize      int
+	MinEvictionAge       time.Duration
+}
+
+// OptionsConstructor contains fields required to initialize an options struct
+// If a field is nil, the corresponding field in the options struct will be set to the default value
+type OptionsConstructor struct {
```

Comment:
Instead of renaming the public type, could we rename the internal type? Options/Opts is a common convention in Go for passing options to initialize a struct. Alternatively we could get rid of the internal options type and inline the fields into the cache struct.

Reply:
Let me know if this is cleaner. Agree that it's confusing to have the internal and public options types, so inlined the options into the cache struct